diff --git a/style.ini b/style.ini
index 77ef564..5a0b409 100644
--- a/style.ini
+++ b/style.ini
@@ -58,7 +58,7 @@ __background__  = "#fff"
 __font_size__   = "1rem"
 __font_family__ = "-apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, \"Helvetica Neue\", Arial, sans-serif, \"Apple Color Emoji\", \"Segoe UI Emoji\", \"Segoe UI Symbol\""
 __line_height__ = "1.5"
-__site_width__  = "100%"
+__site_width__  = "65%"
 
 __admin_background_color__  = "#f2f2f2"
 
