diff --git a/main.php b/main.php
index adab022..392d58d 100644
--- a/main.php
+++ b/main.php
@@ -38,7 +38,7 @@ $htmlLang = ' lang="' . $conf['lang'] . ( $lang['direction'] != 'ltr' ? '" dir="
 			<div id="siteLogo">
 				<?php my_sitelogo(); ?>
 
-				<h2 class="title"><?php tpl_link( my_homelink(), htmlentities($conf['title']), ''); ?></h2>
+				<h2 class="title" style="display: none !important;"><?php tpl_link( my_homelink(), htmlentities($conf['title']), ''); ?></h2>
 				<p class="claim"><?php echo $conf['tagline']; ?></p>
 			</div>
 			<div id="globalTools">
@@ -120,4 +120,4 @@ $htmlLang = ' lang="' . $conf['lang'] . ( $lang['direction'] != 'ltr' ? '" dir="
 	<div class="no"><?php tpl_indexerWebBug() /* provide DokuWiki housekeeping, required in all templates */ ?></div>
 	<div id="screen__mode" class="no"></div><?php /* helper to detect CSS media query in script.js */ ?>
 </body>
-</html>
\ No newline at end of file
+</html>
