diff --git a/config.sh b/config.sh
index b1e0374..52afee7 100755
--- a/config.sh
+++ b/config.sh
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 
-USER_CONFIG="$HOME/.config/sketchybar/userconfig.sh"
+USER_CONFIG="$SKETCHYBAR_DIR/userconfig.sh"
 test -f $USER_CONFIG && source $USER_CONFIG
 
 PADDINGS=3
diff --git a/item.sh b/item.sh
index c6d750f..db84007 100755
--- a/item.sh
+++ b/item.sh
@@ -1,31 +1,31 @@
 #!/usr/bin/env sh
 
-sketchybar --add event    dynamic_island_queue \
-		   --add event	  dynamic_island_request \
-		   --add item     island center               \
-           --set island   width=$P_DYNAMIC_ISLAND_DEFAULT_WIDTH         \
-                          background.height=50         \
-						  background.y_offset=9         \
-						  background.color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
-						  background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
-						  background.drawing=true	\
-						  background.padding_left=0 \
-						  background.padding_right=0 \
-						  update_freq=2					\
-						  mach_helper=git.crissnb.islandhelper \
-						  drawing=on				\
-						  popup.background.height=30 \
-						  popup.height=$P_DYNAMIC_ISLAND_DEFAULT_HEIGHT \
-						  popup.align=center \
-						  popup.y_offset=-69 \
-						  popup.horizontal=on \
-						  popup.background.border_color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
-						  popup.background.color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
-						  popup.background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
-						  popup.background.padding_left=0 \
-						  popup.background.padding_right=0 \
-						  popup.background.shadow.drawing=off \
-						  popup.drawing=false
+sketchybar --add event dynamic_island_queue \
+	--add event dynamic_island_request \
+	--add item island center \
+	--set island width=$P_DYNAMIC_ISLAND_DEFAULT_WIDTH \
+	background.height=50 \
+	background.y_offset=9 \
+	background.color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
+	background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
+	background.drawing=true \
+	background.padding_left=0 \
+	background.padding_right=0 \
+	update_freq=2 \
+	mach_helper=git.crissnb.islandhelper \
+	drawing=on \
+	popup.background.height=30 \
+	popup.height=$P_DYNAMIC_ISLAND_DEFAULT_HEIGHT \
+	popup.align=center \
+	popup.y_offset=-69 \
+	popup.horizontal=on \
+	popup.background.border_color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
+	popup.background.color=$P_DYNAMIC_ISLAND_COLOR_BLACK \
+	popup.background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
+	popup.background.padding_left=0 \
+	popup.background.padding_right=0 \
+	popup.background.shadow.drawing=off \
+	popup.drawing=false
 
 if [[ $P_DYNAMIC_ISLAND_DISPLAY == "Primary" ]]; then
 	sketchybar --set island associated_display=1
@@ -35,7 +35,7 @@ fi
 
 # subscribe to events to communicate with helper
 sketchybar --subscribe island dynamic_island_queue \
-		   --subscribe island dynamic_island_request
+	--subscribe island dynamic_island_request
 
 # module initalization
 if [[ $P_DYNAMIC_ISLAND_MUSIC_ENABLED == 1 ]]; then
@@ -49,35 +49,35 @@ if [[ $P_DYNAMIC_ISLAND_MUSIC_ENABLED == 1 ]]; then
 
 	source "$DYNAMIC_ISLAND_DIR/scripts/islands/music/creator.sh"
 
-	sketchybar --add event	  music_change $MUSIC_EVENT \
-			   --add item	  musicListener center \
-			   --set musicListener	script="$DYNAMIC_ISLAND_DIR/scripts/islands/music/handler.sh $P_DYNAMIC_ISLAND_MUSIC_SOURCE" \
-			   						width=0 \
-			   --subscribe musicListener music_change
+	sketchybar --add event music_change $MUSIC_EVENT \
+		--add item musicListener center \
+		--set musicListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/music/handler.sh $P_DYNAMIC_ISLAND_MUSIC_SOURCE" \
+		width=0 \
+		--subscribe musicListener music_change
 fi
 
 if [[ $P_DYNAMIC_ISLAND_APPSWITCH_ENABLED == 1 ]]; then
 	source "$DYNAMIC_ISLAND_DIR/scripts/islands/appswitch/creator.sh"
-	sketchybar --add item	  frontAppSwitchListener center \
-			   --set frontAppSwitchListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/appswitch/handler.sh" \
-			   								width=0 \
-			   --subscribe frontAppSwitchListener front_app_switched
+	sketchybar --add item frontAppSwitchListener center \
+		--set frontAppSwitchListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/appswitch/handler.sh" \
+		width=0 \
+		--subscribe frontAppSwitchListener front_app_switched
 fi
 
 if [[ $P_DYNAMIC_ISLAND_VOLUME_ENABLED == 1 ]]; then
 	source "$DYNAMIC_ISLAND_DIR/scripts/islands/volume/creator.sh"
 	sketchybar --add item volumeChangeListener center \
-			   --set volumeChangeListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/volume/handler.sh" \
-			   								width=0 \
-			   --subscribe volumeChangeListener volume_change
+		--set volumeChangeListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/volume/handler.sh" \
+		width=0 \
+		--subscribe volumeChangeListener volume_change
 fi
 
 if [[ $P_DYNAMIC_ISLAND_BRIGHTNESS_ENABLED == 1 ]]; then
 	source "$DYNAMIC_ISLAND_DIR/scripts/islands/brightness/creator.sh"
 	sketchybar --add item brightnessChangeListener center \
-			   --set brightnessChangeListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/brightness/handler.sh" \
-			   								width=0 \
-			   --subscribe brightnessChangeListener brightness_change
+		--set brightnessChangeListener script="$DYNAMIC_ISLAND_DIR/scripts/islands/brightness/handler.sh" \
+		width=0 \
+		--subscribe brightnessChangeListener brightness_change
 fi
 
 if [[ $P_DYNAMIC_ISLAND_NOTIFICATION_ENABLED == 1 ]]; then
diff --git a/listener.sh b/listener.sh
index 886b7ac..97822fe 100755
--- a/listener.sh
+++ b/listener.sh
@@ -1,7 +1,7 @@
 #!/usr/bin/env sh
 
 sketchybar --add item di_helper_listener center \
-		   --add event di_helper_listener_event \
-		   --subscribe di_helper_listener di_helper_listener_event \
-		   --set di_helper_listener script="$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/process.sh" \
-			   								width=0
+	--add event di_helper_listener_event \
+	--subscribe di_helper_listener di_helper_listener_event \
+	--set di_helper_listener script="$DYNAMIC_ISLAND_DIR/process.sh" \
+	width=0
diff --git a/process.sh b/process.sh
index a074ba6..90c227b 100755
--- a/process.sh
+++ b/process.sh
@@ -4,45 +4,48 @@
 # $OVERRIDE - whether or not to override the existing dynamic island element
 # $ARGS - necessary arguments to activate the dynamic island element
 
-ISLAND_DIR="$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands"
+ISLAND_DIR="$DYNAMIC_ISLAND_DIR/scripts/islands"
+
+# echo "ISLAND_DIR=$ISLAND_DIR"
+# echo "IDENTIFIER=$IDENTIFIER"
 
 case "$IDENTIFIER" in
-	"appswitch")
+"appswitch")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/appswitch/appswitch_island.sh"
 	fi
 	# arg: app name
 	bash "$ISLAND_DIR/appswitch/appswitch_island.sh" "$OVERRIDE|$ARGS"
 	;;
-	"volume")
+"volume")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/volume/volume_island.sh"
 	fi
 	# arg: volume amt
 	bash "$ISLAND_DIR/volume/volume_island.sh" "$OVERRIDE|$ARGS"
 	;;
-	"brightness")
+"brightness")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/brightness/brightness_island.sh"
 	fi
 	# arg: volume amt
 	bash "$ISLAND_DIR/brightness/brightness_island.sh" "$OVERRIDE|$ARGS"
 	;;
-	"pause")
+"pause")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/music/pause_island.sh"
 	fi
 	# arg: pause status
 	bash "$ISLAND_DIR/music/pause_island.sh" "$OVERRIDE|$ARGS"
 	;;
-	"music")
+"music")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/music/music_island.sh"
 	fi
 	# arg: none
 	bash "$ISLAND_DIR/music/music_island.sh" "$OVERRIDE"
 	;;
-	"notifications")
+"notifications")
 	if [[ $OVERRIDE == 1 ]]; then
 		pkill -f "$ISLAND_DIR/notification/notification_island.sh"
 	fi
diff --git a/scripts/islands/appswitch/appswitch_island.sh b/scripts/islands/appswitch/appswitch_island.sh
index c050148..3586147 100755
--- a/scripts/islands/appswitch/appswitch_island.sh
+++ b/scripts/islands/appswitch/appswitch_island.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
+source "$SKETCHYBAR_DIR/userconfig.sh"
 
 SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_APPSWITCH_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 MAX_EXPAND_HEIGHT=$(($P_DYNAMIC_ISLAND_APPSWITCH_EXPAND_HEIGHT+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
@@ -61,4 +61,4 @@ sketchybar --animate tanh 20 --set island popup.height=$P_DYNAMIC_ISLAND_DEFAULT
 
 sleep 0.4
 
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/appswitch/reset.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/appswitch/reset.sh"
diff --git a/scripts/islands/brightness/brightness_island.sh b/scripts/islands/brightness/brightness_island.sh
index 8c6b511..ebabd35 100755
--- a/scripts/islands/brightness/brightness_island.sh
+++ b/scripts/islands/brightness/brightness_island.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
+source "$SKETCHYBAR_DIR/userconfig.sh"
 
 SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_BRIGHTNESS_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_BRIGHTNESS_MAX_EXPAND_WIDTH+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
@@ -77,4 +77,4 @@ sketchybar --animate tanh 20 --set island popup.height=$P_DYNAMIC_ISLAND_BRIGHTN
 
 sleep 0.5
 
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/brightness/reset.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/brightness/reset.sh"
diff --git a/scripts/islands/music/handler.sh b/scripts/islands/music/handler.sh
index 540a99b..79eefb7 100755
--- a/scripts/islands/music/handler.sh
+++ b/scripts/islands/music/handler.sh
@@ -12,19 +12,19 @@ if [ "$(echo "$INFO" | jq -r '.["Player State"]')" = "Stopped" ]; then
 	exit 0
 fi
 
-cache="$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/music/data/cache"
+cache="/tmp/dynamic-island-cache"
 PLAYER_STATE=$(osascript -e "tell application \"$1\" to return (get player state)")
 
 if [[ $(cat $cache) == 0 ]]; then
 	# resume
-	printf 1 > "$cache"
+	printf 1 >"$cache"
 	sketchybar --trigger dynamic_island_queue INFO="pause" ISLAND_ARGS="1"
 	exit 0
 fi
 
 if [[ $PLAYER_STATE == "paused" ]]; then
 	# paused
-	printf 0 > "$cache"
+	printf 0 >"$cache"
 	sketchybar --trigger dynamic_island_queue INFO="pause" ISLAND_ARGS="0"
 	exit 0
 fi
diff --git a/scripts/islands/music/music_island.sh b/scripts/islands/music/music_island.sh
index 2cc8fd9..bde1aa2 100755
--- a/scripts/islands/music/music_island.sh
+++ b/scripts/islands/music/music_island.sh
@@ -1,10 +1,10 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
-ARTWORK_LOCATION="$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/music/artwork.jpg"
+source "$SKETCHYBAR_DIR/userconfig.sh"
+ARTWORK_LOCATION="$DYNAMIC_ISLAND_DIR/scripts/islands/music/artwork.jpg"
 
-INFO_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
-INFO_MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_MAX_EXPAND_WIDTH+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
-INFO_MAX_EXPAND_HEIGHT=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
+INFO_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_WIDTH - $P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
+INFO_MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_MAX_EXPAND_WIDTH + $P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
+INFO_MAX_EXPAND_HEIGHT=$(($P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT + $P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 
 # $1 - override
 override=$1
@@ -14,7 +14,7 @@ TITLE=$(osascript -e "tell application \"$P_DYNAMIC_ISLAND_MUSIC_SOURCE\" to get
 ARTIST=$(osascript -e "tell application \"$P_DYNAMIC_ISLAND_MUSIC_SOURCE\" to get artist of current track")
 
 if [[ $P_DYNAMIC_ISLAND_MUSIC_SOURCE == "Music" ]]; then
-	osascript "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/music/get_artwork.scpt"
+	osascript "$DYNAMIC_ISLAND_DIR/scripts/islands/music/get_artwork.scpt"
 elif [[ $P_DYNAMIC_ISLAND_MUSIC_SOURCE == "Spotify" ]]; then
 	COVER=$(osascript -e 'tell application "Spotify" to get artwork url of current track')
 	curl -s --max-time 25 "$COVER" -o "$ARTWORK_LOCATION"
@@ -23,54 +23,54 @@ else
 fi
 
 if [[ ${#TITLE} -gt 25 ]]; then
-  TITLE=$(printf "$(echo $TITLE | cut -c 1-25)…")
+	TITLE=$(printf "$(echo $TITLE | cut -c 1-25)…")
 fi
 
 if [[ ${#ARTIST} -gt 25 ]]; then
-  ARTIST=$(printf "$(echo $ARTIST | cut -c 1-25)…")
+	ARTIST=$(printf "$(echo $ARTIST | cut -c 1-25)…")
 fi
 
 if [[ $override == "0" ]]; then
 	sketchybar --set island.music_artwork drawing=on \
-			   --set island.music_title drawing=on \
-			   --set island.music_artist drawing=on \
-			   --set island.music_placeholder drawing=on \
-			   --set island popup.drawing=true \
-			                background.drawing=false \
-							popup.horizontal=on \
-							popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_DEFAULT_HEIGHT
+		--set island.music_title drawing=on \
+		--set island.music_artist drawing=on \
+		--set island.music_placeholder drawing=on \
+		--set island popup.drawing=true \
+		background.drawing=false \
+		popup.horizontal=on \
+		popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_DEFAULT_HEIGHT
 fi
 
 sketchybar --set island.music_artist label="$ARTIST" \
-	       --set island.music_title label="$TITLE" \
-		   --set island.music_artwork background.image="$ARTWORK_LOCATION"
+	--set island.music_title label="$TITLE" \
+	--set island.music_artwork background.image="$ARTWORK_LOCATION"
 
 if [[ $override == "0" ]]; then
 	sketchybar --animate sin 15 --set island.music_placeholder width=$INFO_SQUISH_WIDTH width=$INFO_MAX_EXPAND_SQUISH_WIDTH width=$P_DYNAMIC_ISLAND_MUSIC_INFO_MAX_EXPAND_WIDTH \
-			   --animate sin 25 --set island popup.height=$INFO_MAX_EXPAND_HEIGHT popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT \
-			   --animate sin 20 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_MUSIC_INFO_CORNER_RAD
+		--animate sin 25 --set island popup.height=$INFO_MAX_EXPAND_HEIGHT popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT \
+		--animate sin 20 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_MUSIC_INFO_CORNER_RAD
 else
 	sketchybar --animate sin 15 --set island.music_placeholder width=$INFO_MAX_EXPAND_SQUISH_WIDTH width=$P_DYNAMIC_ISLAND_MUSIC_INFO_MAX_EXPAND_WIDTH \
-			   --animate sin 25 --set island popup.height=$INFO_MAX_EXPAND_HEIGHT popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT \
-			   --animate sin 20 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_MUSIC_INFO_CORNER_RAD
+		--animate sin 25 --set island popup.height=$INFO_MAX_EXPAND_HEIGHT popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_HEIGHT \
+		--animate sin 20 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_MUSIC_INFO_CORNER_RAD
 fi
 
 sleep 0.15
 sketchybar --animate tanh 25 --set island.music_title label.color=$P_DYNAMIC_ISLAND_COLOR_WHITE \
-		   --animate tanh 25 --set island.music_artist label.color=$P_DYNAMIC_ISLAND_COLOR_WHITE \
-		   --animate tanh 25 --set island.music_artwork background.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT
+	--animate tanh 25 --set island.music_artist label.color=$P_DYNAMIC_ISLAND_COLOR_WHITE \
+	--animate tanh 25 --set island.music_artwork background.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT
 
 sleep 1.5
 
 sketchybar --animate tanh 25 --set island.music_title label.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT \
-		   --animate tanh 25 --set island.music_artist label.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT \
-		   --animate tanh 25 --set island.music_artwork background.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT
+	--animate tanh 25 --set island.music_artist label.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT \
+	--animate tanh 25 --set island.music_artwork background.color=$P_DYNAMIC_ISLAND_COLOR_TRANSPARENT
 
 sleep 0.4
 sketchybar --animate tanh 25 --set island popup.height=$P_DYNAMIC_ISLAND_MUSIC_INFO_DEFAULT_HEIGHT \
-		   --animate  sin 30 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
-		   --animate tanh 20 --set island.music_placeholder width=$INFO_SQUISH_WIDTH width=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_WIDTH
+	--animate sin 30 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS \
+	--animate tanh 20 --set island.music_placeholder width=$INFO_SQUISH_WIDTH width=$P_DYNAMIC_ISLAND_MUSIC_INFO_EXPAND_WIDTH
 
 sleep 0.7
 
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/music/reset.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/music/reset.sh"
diff --git a/scripts/islands/music/pause_island.sh b/scripts/islands/music/pause_island.sh
index c7c54ae..d484596 100755
--- a/scripts/islands/music/pause_island.sh
+++ b/scripts/islands/music/pause_island.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
+source "$SKETCHYBAR_DIR/userconfig.sh"
 
 RESUME_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_RESUME_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 RESUME_MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_MUSIC_RESUME_MAX_EXPAND_WIDTH+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
@@ -51,4 +51,4 @@ sketchybar --animate tanh 20 --set island popup.height=$P_DYNAMIC_ISLAND_DEFAULT
 
 sleep 0.7
 
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/music/reset-resume.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/music/reset-resume.sh"
diff --git a/scripts/islands/notification/notification_island.sh b/scripts/islands/notification/notification_island.sh
index 0946b63..52a47c5 100755
--- a/scripts/islands/notification/notification_island.sh
+++ b/scripts/islands/notification/notification_island.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
+source "$SKETCHYBAR_DIR/userconfig.sh"
 
 SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_NOTIFICATION_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_NOTIFICATION_MAX_EXPAND_WIDTH+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
@@ -61,4 +61,4 @@ sketchybar --animate tanh 25 --set island popup.background.corner_radius=$P_DYNA
 
 sleep 0.7
 
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/notification/reset.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/notification/reset.sh"
diff --git a/scripts/islands/volume/volume_island.sh b/scripts/islands/volume/volume_island.sh
index a071b26..eb86a3a 100755
--- a/scripts/islands/volume/volume_island.sh
+++ b/scripts/islands/volume/volume_island.sh
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
-source "$HOME/.config/sketchybar/userconfig.sh"
+source "$SKETCHYBAR_DIR/userconfig.sh"
 
 SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_VOLUME_EXPAND_WIDTH-$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
 MAX_EXPAND_SQUISH_WIDTH=$(($P_DYNAMIC_ISLAND_VOLUME_MAX_EXPAND_WIDTH+$P_DYNAMIC_ISLAND_SQUISH_AMOUNT))
@@ -77,4 +77,4 @@ sketchybar --animate tanh 20 --set island popup.height=$P_DYNAMIC_ISLAND_VOLUME_
 		   --animate sin 25 --set island popup.background.corner_radius=$P_DYNAMIC_ISLAND_DEFAULT_CORNER_RADIUS
 
 sleep 0.5
-source "$HOME/.config/sketchybar/plugins/Dynamic-Island-Sketchybar/scripts/islands/volume/reset.sh"
+source "$DYNAMIC_ISLAND_DIR/scripts/islands/volume/reset.sh"
